AEPIdentity/Sources/IdentityState.swift

```diff
@@ -49,6 +49,7 @@ class IdentityState {
 
         // load data from local storage
         identityProperties.loadFromPersistence()
+        Log.debug(label: "\(LOG_TAG):\(#function)", "Load ECID from persistence: ecid = \(identityProperties.ecid?.ecidString ?? "nil")")
```

Comment:
nit: can use trace

Reply:
I've updated this log message to match the one used in Android, which is why it is logging more than previously.